.github/workflows/build.yml

```
@@ -7,11 +7,12 @@
 
 name: build
 
-on:
-  push:
-    branches: [ master ]
-  pull_request:
-    branches: [ master ]
+on: push
+  # push:
+  #   branches: [ master ]
+  # pull_request:
+  #   branches: [ master ]
```

So do we want:

```suggestion
on:
  pull_request:
  push:
    branches: [ master ]
```

Docs say that `pull_request` by default will be triggered on event types `opened`, `synchronize` or `reopened`, so looks about right.

---

Yes sounds good